docs/articles/multi-attribute-semantic-search.md

```diff
@@ -0,0 +1,459 @@
+| Dreamweaver Octopus | 0.402288 | Cephalopod with tentacles that shimmer like auroras                  | Deep ocean trenches and underwater caves | Influences the dreams of nearby creatures                      |
+| Aqua Wraith         | 0.330869 | Translucent humanoid figure made of flowing water                    | Rivers, lakes, and coastal areas         | Shapeshifts to blend with water bodies and controls currents   |
+
+The top result is the most relevant one, the "Harmonic Coral" itself which is pretty much expected.
+Let's see who are the other two monsters: "Dreamweaver Octopus" and "Aqua Wraith".
+
+Both share important thematic elements with the Harmonic Coral:
+1. Aquatic habitats
+2. Ability to influence or manipulate their environment
+3. Dynamic or fluid visual characteristics
+These common themes in habitat, behavior, and appearance explain why these monsters were identified as most similar to the Harmonic Coral.
+
+### Attribute weighing
+
+Let's suppose that we want to give more importance to the "look" attribute.
+It's easy to do with Superlinked.
+All we need to do is to add weights to the query.
+
+```python
+monster_query_with_weights = (
+    Query(monster_index, weights={
+        look_space: Param("look_weight"),
+        habitat_space: Param("habitat_weight"),
+        behavior_space: Param("behavior_weight")
+    })
+    .find(monster)
+    .similar(look_space.text, Param("look"))
+    .similar(habitat_space.text, Param("habitat"))
+    .similar(behavior_space.text, Param("behavior"))
+    .limit(LIMIT)
+)
+```
+
+Let's again search for similar monsters to the "Harmonic Coral", but this time use weights for the attributes.
+First of all, let's find monsters with the most close appearance:
+
+```python
+weights = {
+    "look_weight": 1.0,
+    "habitat_weight": 0,
+    "behavior_weight": 0
+}
+```
+
+| id                  |    score | look                                                                 | habitat                            | behavior                                                       |
+|:--------------------|---------:|:---------------------------------------------------------------------|:-----------------------------------|:---------------------------------------------------------------|
+| Harmonic Coral      | 0.57735  | Branching, musical instrument-like structure with vibrating tendrils | Shallow seas and tidal pools       | Creates complex melodies to communicate and influence emotions |
+| Thornvine Elemental | 0.252593 | Plant-like creature with a body of twisted vines and thorns          | Overgrown ruins and dense jungles  | Rapidly grows and controls surrounding plant life              |
+| Plasma Serpent      | 0.243241 | Snake-like creature made of crackling energy                         | Electrical storms and power plants | Feeds on electrical currents and can short-circuit technology  |
+
+**WARNING! Score is not 1 for the first result! It equals 1 / sqrt(3)**
```

Comment:
this is something the [Query arithmetics project](https://linear.app/superlinked/project/query-arithmetics-79e40faa4a70/overview) will address.